packages/core/android/src/main/java/com/segmentanalyticsreactnative/AnalyticsReactNativePackage.kt

```diff
@@ -10,11 +10,29 @@ import com.facebook.react.uimanager.ViewManager
 import com.facebook.react.bridge.JavaScriptModule
 
 class AnalyticsReactNativePackage : ReactPackage {
+
+    private var isInitialized = false
+    private var anonymousId = ""
```

Comment:
Don't make it default to `""` as it will send it as an empty string every time this module starts.
A better idea would be to make this nullable and check if it is set to anything else than null before sending. That should save bridge calls and also prevent resetting things on the RN side (or having an explicit check for `""` which is pretty brittle)